packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-widget.tsx

```diff
@@ -48,16 +60,18 @@ const ContractTypeWidget = ({ is_equal, name, value, list, onChange, languageCha
         }
     };
 
-    const handleInfoClick = clicked_item => {
+    React.useEffect(() => {
```

Comment:
@henry-deriv we got rid of this useEffect in [Turbos PR](https://github.com/binary-com/deriv-app/pull/7514/files) :) we don't need it anymore. [Exact commit, please find this file in there ](https://github.com/maryia-deriv/deriv-app/commit/b7f1172245c1920c441fd657c65d8d63f63e7c3e#diff-70dd541692596463ae92de8ec8f57b97d2c954abd00932d345f9b69fa546e257)

Reply:
yes. I asked @kate-deriv and she doesnt remember why it was removed. I opted to keep it because it was working fine prior. Just to be safe

Reply:
It was some fix, but I don't remember what exactly because it was too long ago

Reply:
@henry-deriv It's ok, I'll remind you @kate-deriv :) lol. @kate-deriv and I decided to remove it on Turbos PR because it's unnecessary, there is no need to have this useEffect. The component works very well without it, and QA passed.